cartography/intel/aws/ec2.py

```diff
@@ -496,6 +536,129 @@ def load_ec2_auto_scaling_groups(neo4j_session, data, region, current_aws_accoun
                 )
 
 
+def load_load_balancer_v2s(neo4j_session, data, region, current_aws_account_id, aws_update_tag):
+    """
+    For each elbv2
+        'LoadBalancers': [
```

Comment:
This comment showing the expected data type is helpful, but can you please add  integration tests for `load_load_balancer_v2s()`, `load_load_balancer_v2_subnets()`,  `load_load_balancer_v2_target_groups()`, and `load_load_balancer_v2_listeners()`?

In general we just need tests for the `load_*` functions and you can use the mock data you've specified here already.  A good place to copy from is here: https://github.com/lyft/cartography/blob/master/tests/integration/cartography/intel/aws/test_dynamodb.py

Reply:
Integration tests have been written for all four `load_load_balancer_v2_...` methods.